mithril-explorer/components/EpochSettings/index.js

```diff
@@ -0,0 +1,57 @@
+import React, {useEffect, useState} from 'react';
+import {Card, ListGroup} from "react-bootstrap";
+import RawJsonButton from "../RawJsonButton";
+import {useSelector} from "react-redux";
+
+export default function EpochSettings(props) {
+  const [epochSettings, setEpochSettings] = useState({});
+  const aggregator = useSelector((state) => state.settings.selectedAggregator);
+  const autoUpdate = useSelector((state) => state.settings.autoUpdate);
+  const updateInterval = useSelector((state) => state.settings.updateInterval);
+
+  useEffect(() => {
+    if (!autoUpdate) {
+      return;
+    }
+
+    let fetchEpochSettings = () => {
+      fetch(`${aggregator}/epoch-settings`)
+        .then(response => response.status === 200 ? response.json() : {})
+        .then(data => setEpochSettings(data))
+        .catch(error => {
+          setEpochSettings({});
+          console.error("Fetch epoch-settings error:", error);
+        });
+    };
+
+    // Fetch it once without waiting
+    fetchEpochSettings();
+
+    const interval = setInterval(fetchEpochSettings, updateInterval);
+    return () => clearInterval(interval);
+  }, [aggregator, updateInterval, autoUpdate]);
+
+  return (
+    <div>
+      <h2>
+        Epoch Settings
+        <RawJsonButton href={`${aggregator}/epoch-settings`} variant="outline-light" size="sm"/>
+      </h2>
+
+      <Card>
+        <Card.Body>
+          <Card.Title>Current Epoch</Card.Title>
+          <ListGroup variant="flush">
+            <ListGroup.Item>{epochSettings.epoch}</ListGroup.Item>
+          </ListGroup>
+          <Card.Title>Protocol Parameters</Card.Title>
+          <ListGroup horizontal>
+            <ListGroup.Item>K: {epochSettings.protocol?.k}</ListGroup.Item>
+            <ListGroup.Item>M: {epochSettings.protocol?.m}</ListGroup.Item>
+            <ListGroup.Item>Phi: {epochSettings.protocol?.phi_f}</ListGroup.Item>
```

Comment:
```suggestion
            <ListGroup.Item>k: {epochSettings.protocol?.k}</ListGroup.Item>
            <ListGroup.Item>m: {epochSettings.protocol?.m}</ListGroup.Item>
            <ListGroup.Item>f: {epochSettings.protocol?.phi_f}</ListGroup.Item>
```

Reply:
Done for all Params, see: https://github.com/input-output-hk/mithril/pull/590/commits/a33ebfa7580a08ea80c4fdef4bf0a817875fef83